packages/connectivity/src/scp-cf/destination/destination-accessor.ts

```diff
@@ -121,6 +121,11 @@ function createDestinationFetchLogs(
 export async function getAllDestinationsFromDestinationService(
   options: AllDestinationOptions = {}
```

Comment:
```suggestion
  options: AllDestinationOptions = { useCache: true }
```

Tom gave a nice suggestion, this way you dont have to write:

```
options = {
    ...options,
    // Enable caching by default
    useCache: options.useCache ?? true
  };
```

Reply:
So, I had initially added this, but this default is only used, if the user doesn't explicitly provide any `options`.
So, code right now always adds `useCache` option regardless of if the user provided `options` or not.